src/events/pg.js

```diff
@@ -0,0 +1,101 @@
+const uuid4 = require('uuid4');
+const shimmer = require('shimmer');
+const pg = require('pg')
+const sqlParser = require('node-sqlparser');
```

Comment:
This package is not in the package.json, is something missing?

Reply:
now it is.